ocaml/xapi/xapi_bond.ml

```diff
@@ -385,7 +386,7 @@ let create ~__context ~network ~members ~mAC ~mode ~properties =
         ~ipv6_configuration_mode:`None ~iPv6:[""] ~ipv6_gateway:"" ~primary_address_type:`IPv4 ~managed:true
         ~properties:pif_properties ~capabilities:[] ~pCI:Ref.null;
       Db.Bond.create ~__context ~ref:bond ~uuid:(Uuid.to_string (Uuid.make_uuid ())) ~master:master ~other_config:[]
-        ~primary_slave ~mode ~properties ~links_up:0L;
+        ~primary_slave ~mode ~properties ~links_up:0L ~auto_update_mac:auto_update_mac;
```

Comment:
This can be shortened to simply `~auto_update_mac` because both the parameter and the variable have the same name